linter/lookml_linter.py

```diff
@@ -20,6 +20,10 @@ def run(self) -> None:
             file_data = self.data[filename]
             views = file_data.get('views', [])
             explores = file_data.get('explores', [])
+            if len(explores) > 0:
+                includes = file_data.get('includes', [])
+                first_explore = explores[0]
+                first_explore['includes'] = includes
```

Comment:
My rationale here is that I only want to add the includes to one Explore in each file. Another option I considered is adding the includes to each Explore in a file with something like:

```
includes = file_data.get('includes', [])
for e in explores:
	e.update({'includes': includes})
```

However, the drawback with that is that it'll flag wildcard view includes once for every Explore in a file, which seems excessive. I thought it made more sense for any infraction to this rule to only be surfaced once per file.

Let me know if you think that makes sense or think we should approach this another way.

Reply:
Nope! Good idea, but we should treat "includes" as exactly what they are, a separate entity.  They're not _strictly_  associated with explores, as you can include any lkml file from any other file.  We should go with:

```
includes = file_data.get('includes', [])
...
for i in includes:
    self._lint_object({
       'name': i
    }, 'include')
```